apps/sports-helsinki/src/domain/search/landingPageSearch/LandingPageSearch.tsx

```diff
@@ -36,7 +36,7 @@ const Search: React.FC = () => {
       {
         q: [option.text],
         // Order by relevance if a search filter is applied
-        // orderBy: autosuggestInput ? OrderBy.relevance : undefined,
+        // orderBy: autosuggestInput ? UnifiedSearchOrderBy.relevance : undefined,
```

Comment:
This and line above (L28) are modified but commented out, should they be..?

Reply:
I'll probably remove the commented out code and fix the underlying problem that the default sort order used in UnifiedSearch was by name rather than by relevance and [LIIKUNTA-355](https://helsinkisolutionoffice.atlassian.net/browse/LIIKUNTA-355) defined that the default sort order should be by relevance. This could be done e.g. with this:
```diff
diff --git a/apps/sports-helsinki/src/domain/search/eventSearch/searchResultList/UnifiedSearchOrderBySelect.tsx b/apps/sports-helsinki/src/domain/search/eventSearch/searchResultList/UnifiedSearchOrderBySelect.tsx
index a662077e..a1bfaf11 100644
--- a/apps/sports-helsinki/src/domain/search/eventSearch/searchResultList/UnifiedSearchOrderBySelect.tsx
+++ b/apps/sports-helsinki/src/domain/search/eventSearch/searchResultList/UnifiedSearchOrderBySelect.tsx
@@ -19,15 +19,15 @@ const UnifiedSearchOrderBySelect: React.FC = () => {
   const handleUnifiedSearchOrderChange = useHandleUnifiedSearchOrderChange();

   const defaultOption = {
-    text: t('search:orderBy.alphabetical'),
-    value: `${UnifiedSearchOrderBy.name}-${OrderDir.asc}`,
+    text: t('search:orderBy.relevance'),
+    value: `${UnifiedSearchOrderBy.relevance}-${OrderDir.asc}`,
   };
   const orderByOptions = [
-    defaultOption,
     {
-      text: t('search:orderBy.relevance'),
-      value: `${UnifiedSearchOrderBy.relevance}-${OrderDir.asc}`,
+      text: t('search:orderBy.alphabetical'),
+      value: `${UnifiedSearchOrderBy.name}-${OrderDir.asc}`,
     },
+    defaultOption,
     {
       text: t('search:orderBy.distance'),
       value: `${UnifiedSearchOrderBy.distance}-${OrderDir.asc}`,
diff --git a/apps/sports-helsinki/src/domain/search/landingPageSearch/LandingPageSearch.tsx b/apps/sports-helsinki/src/domain/search/landingPageSearch/LandingPageSearch.tsx
index 8ca63b04..f0a45b68 100644
--- a/apps/sports-helsinki/src/domain/search/landingPageSearch/LandingPageSearch.tsx
+++ b/apps/sports-helsinki/src/domain/search/landingPageSearch/LandingPageSearch.tsx
@@ -24,8 +24,6 @@ const Search: React.FC = () => {
     setFilters(
       {
         q: [autosuggestInput],
-        // Order by relevance if a search filter is applied
-        // orderBy: autosuggestInput ? UnifiedSearchOrderBy.relevance : undefined,
       },
       getI18nPath(ROUTES.SEARCH, locale)
     );
@@ -35,8 +33,6 @@ const Search: React.FC = () => {
     setFilters(
       {
         q: [option.text],
-        // Order by relevance if a search filter is applied
-        // orderBy: autosuggestInput ? UnifiedSearchOrderBy.relevance : undefined,
       },
       getI18nPath(ROUTES.SEARCH, locale)
     );
diff --git a/apps/sports-helsinki/src/domain/unifiedSearch/useUnifiedSearchVariables.ts b/apps/sports-helsinki/src/domain/unifiedSearch/useUnifiedSearchVariables.ts
index 7c8d47df..ea6c7664 100644
--- a/apps/sports-helsinki/src/domain/unifiedSearch/useUnifiedSearchVariables.ts
+++ b/apps/sports-helsinki/src/domain/unifiedSearch/useUnifiedSearchVariables.ts
@@ -89,7 +89,7 @@ export default function useUnifiedSearchVariables(
       administrativeDivisionIds,
       isOpenNow,
       openAt,
-      orderBy = UnifiedSearchOrderBy.name,
+      orderBy = UnifiedSearchOrderBy.relevance,
       orderDir = OrderDir.asc,
       ontologyWordIds,
       after,


```

Reply:
The above changes are now in commit 4e2418e8c1d2d6905c703344ff76edd21a571f75